<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Shop</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>981</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Products&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">wdjnv</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_1">
        <attribute name="title">
         <string>Dashboard</string>
        </attribute>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>30</y>
           <width>131</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Welcome Manager!</string>
         </property>
        </widget>
        <widget class="QLabel" name="noTrans">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>120</y>
           <width>201</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="moneyDay">
         <property name="geometry">
          <rect>
           <x>320</x>
           <y>120</y>
           <width>201</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="noItems">
         <property name="geometry">
          <rect>
           <x>570</x>
           <y>120</y>
           <width>201</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Products</string>
        </attribute>
        <widget class="QPushButton" name="searchbarcode_products">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>10</y>
           <width>91</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Filter Barcode</string>
         </property>
        </widget>
        <widget class="QPushButton" name="searchname_products">
         <property name="geometry">
          <rect>
           <x>520</x>
           <y>10</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Filter Name</string>
         </property>
        </widget>
        <widget class="QTableWidget" name="tableWidget_products">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>821</width>
           <height>451</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="viewall_products">
         <property name="geometry">
          <rect>
           <x>750</x>
           <y>10</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_2_products">
         <property name="geometry">
          <rect>
           <x>330</x>
           <y>10</y>
           <width>171</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_1_products">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>171</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Perishable Products</string>
        </attribute>
        <widget class="QPushButton" name="searchbarcode_perishable">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>40</y>
           <width>91</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Filter Barcode</string>
         </property>
        </widget>
        <widget class="QPushButton" name="searchname_perishable">
         <property name="geometry">
          <rect>
           <x>530</x>
           <y>40</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Filter Name</string>
         </property>
        </widget>
        <widget class="QTableWidget" name="tableWidget_perishable">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>821</width>
           <height>421</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="viewall_perishable">
         <property name="geometry">
          <rect>
           <x>760</x>
           <y>40</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_2_perishable">
         <property name="geometry">
          <rect>
           <x>340</x>
           <y>40</y>
           <width>171</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_1_perishable">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>171</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="add_perishable">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>111</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Add New Product</string>
         </property>
        </widget>
        <widget class="QPushButton" name="stock_perishable">
         <property name="geometry">
          <rect>
           <x>720</x>
           <y>10</y>
           <width>111</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Restock Product</string>
         </property>
        </widget>
        <widget class="QPushButton" name="remove_perishable">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>10</y>
           <width>111</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Delete Product</string>
         </property>
        </widget>
        <widget class="QPushButton" name="update_perishable">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>10</y>
           <width>111</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Update Product</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_4">
        <attribute name="title">
         <string>Transactions</string>
        </attribute>
        <widget class="QPushButton" name="searchid_tran">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>50</y>
           <width>121</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Filter Transaction ID</string>
         </property>
        </widget>
        <widget class="QPushButton" name="searchdate_tran">
         <property name="geometry">
          <rect>
           <x>540</x>
           <y>50</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Filter Date</string>
         </property>
        </widget>
        <widget class="QTableWidget" name="tableWidget_tran">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>821</width>
           <height>421</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="viewall_tran">
         <property name="geometry">
          <rect>
           <x>760</x>
           <y>50</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_2_tran">
         <property name="geometry">
          <rect>
           <x>380</x>
           <y>50</y>
           <width>141</width>
           <height>20</height>
          </rect>
         </property>
         <property name="placeholderText">
          <string>yyyy-mm-dd</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_1_tran">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>171</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="add_tran">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>131</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Perform A Transaction</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_5">
        <attribute name="title">
         <string>Display Units</string>
        </attribute>
        <widget class="QPushButton" name="searchbarcode_units">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>50</y>
           <width>91</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Filter Barcode</string>
         </property>
        </widget>
        <widget class="QPushButton" name="searchdev_units">
         <property name="geometry">
          <rect>
           <x>530</x>
           <y>50</y>
           <width>101</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Filter Device ID</string>
         </property>
        </widget>
        <widget class="QTableWidget" name="tableWidget_cashier">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>821</width>
           <height>201</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="viewall_units">
         <property name="geometry">
          <rect>
           <x>760</x>
           <y>50</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_2_units">
         <property name="geometry">
          <rect>
           <x>360</x>
           <y>50</y>
           <width>151</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_1_units">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>171</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QTableWidget" name="tableWidget_pdu">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>320</y>
           <width>821</width>
           <height>201</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="newunit">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>141</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Add New Unit</string>
         </property>
        </widget>
        <widget class="QPushButton" name="removeunit">
         <property name="geometry">
          <rect>
           <x>690</x>
           <y>10</y>
           <width>141</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Remove Existing Unit</string>
         </property>
        </widget>
        <widget class="QPushButton" name="updateunit">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>10</y>
           <width>141</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Update Existing Unit</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_6">
        <attribute name="title">
         <string>Promotions</string>
        </attribute>
        <widget class="QPushButton" name="searchbarcode_promo">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>50</y>
           <width>91</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Filter Barcode</string>
         </property>
        </widget>
        <widget class="QPushButton" name="searchname_promo">
         <property name="geometry">
          <rect>
           <x>620</x>
           <y>50</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Filter Name</string>
         </property>
        </widget>
        <widget class="QTableWidget" name="tableWidget_promo">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>821</width>
           <height>421</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="viewall_promo">
         <property name="geometry">
          <rect>
           <x>760</x>
           <y>50</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_3_promo">
         <property name="geometry">
          <rect>
           <x>500</x>
           <y>50</y>
           <width>111</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_1_promo">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>121</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="newpromo">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>131</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Create New Promotion</string>
         </property>
        </widget>
        <widget class="QPushButton" name="removepromo">
         <property name="geometry">
          <rect>
           <x>710</x>
           <y>10</y>
           <width>121</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Delete Promotion</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_2_promo">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>50</y>
           <width>111</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="searchpromo_promo">
         <property name="geometry">
          <rect>
           <x>380</x>
           <y>50</y>
           <width>91</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Filter Promo ID</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>270</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>100</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <property name="verticalSpacing">
          <number>15</number>
         </property>
         <item row="0" column="0">
          <widget class="QPushButton" name="writeoff">
           <property name="styleSheet">
            <string notr="true">QPushButton#writeoff {
background :rgb(197, 115, 255);
color: rgb(255, 255, 255)
}</string>
           </property>
           <property name="text">
            <string>Write Off</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="orderproducts">
           <property name="text">
            <string>Order new Products</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="sendsheets">
           <property name="text">
            <string>Send Balance Sheets</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="updatestock">
           <property name="text">
            <string>Sync With HQ</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="logout">
         <property name="text">
          <string>Log Out</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>logout</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>727</x>
     <y>543</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
